codalab/apps/web/static/js/jquery-eu-cookie-law-popup.js

```diff
@@ -0,0 +1,266 @@
+				 	(_self.params.buttonLearnmoreOpenInNewWindow ? ' target=_blank ' : '') +
+					' class="eupopup-button eupopup-button_2">' + _self.params.buttonLearnmoreTitle + '</a>' +
+				  '<div class="clearfix"></div>' +
+				'</div>' +
+				'<a href="#" class="eupopup-closebutton">x</a>' +
+			'</div>';
+
+		return html;
+	};
+
+	// Storing the consent in a cookie
+	var setUserAcceptsCookies = function(consent) {
+		var d = new Date();
+		var expiresInDays = _self.params.agreementExpiresInDays * 24 * 60 * 60 * 1000;
+		d.setTime( d.getTime() + expiresInDays );
+		var expires = "expires=" + d.toGMTString();
+		document.cookie = _self.vars.COOKIE_NAME + '=' + consent + "; " + expires + ";path=/";
+
+		$(document).trigger("user_cookie_consent_changed", {'consent' : consent});
+	};
+
+	// Let's see if we have a consent cookie already
+	var userAlreadyAcceptedCookies = function() {
+		var userAcceptedCookies = false;
+		var cookies = document.cookie.split(";");
+		for (var i = 0; i < cookies.length; i++) {
+			var c = cookies[i].trim();
+			if (c.indexOf(_self.vars.COOKIE_NAME) == 0) {
+				userAcceptedCookies = c.substring(_self.vars.COOKIE_NAME.length + 1, c.length);
+			}
+		}
+
+		return userAcceptedCookies;
+	};
+	
+	var hideContainer = function() {
+		// $('.eupopup-container').slideUp(200);
+		$('.eupopup-container').animate({
+			opacity: 0,
+			height: 0
+		}, 200, function() {
+			$('.eupopup-container').hide(0);
+		});
+	};
+
+	///////////////////////////////////////////////////////////////////////////////////////////////
+	// PUBLIC FUNCTIONS  //////////////////////////////////////////////////////////////////////////
+	var publicfunc = {
+
+		// INITIALIZE EU COOKIE LAW POPUP /////////////////////////////////////////////////////////
+		init : function(settings) {
+
+			parseParameters(
+				$(".eupopup").first(),
+				$(".eupopup-markup").html(),
+				settings);
+
+			// No need to display this if user already accepted the policy
+			if (userAlreadyAcceptedCookies()) {
+				return;
+			}
+
+			// We should initialise only once
+			if (_self.vars.INITIALISED) {
+				return;
+			}
+			_self.vars.INITIALISED = true;
+
+			// Markup and event listeners >>>
+			_self.vars.HTML_MARKUP = createHtmlMarkup();
+
+			if ($('.eupopup-block').length > 0) {
+				$('.eupopup-block').append(_self.vars.HTML_MARKUP);
+			} else {
+				$('BODY').append(_self.vars.HTML_MARKUP);
+			}
+
+			$('.eupopup-button_1').click(function() {
+				setUserAcceptsCookies(true);
+				hideContainer();
+				return false;
+			});
+			$('.eupopup-closebutton').click(function() {
+				setUserAcceptsCookies(true);
+				hideContainer();
+				return false;
+			});
+			// ^^^ Markup and event listeners
+
+			// Ready to start!
+			$('.eupopup-container').show();
+
+			// In case it's alright to just display the message once 
+			if (_self.params.autoAcceptCookiePolicy) {
+				setUserAcceptsCookies(true);
+			}
+
+		}
+
+	};
+
+	return publicfunc;
+});
+
+$(document).ready( function() {
```

Comment:
Will you move this code out of this vendor library file somewhere else? Like bottom of `base.html` or something?